apps/docs/src/app/platform/component-docs/platform-forms/platform-date-picker/platform-date-picker-examples/platform-date-picker-example.component.html

```diff
@@ -27,7 +28,7 @@ <h3>Date Picker with Reactive Form.</h3>
     <fdp-form-field #ffl5 id="disableddate" column="1" rank="2" required="true" label="Disabled Date Picker:"
         disabled="true">
         <fdp-date-picker name="disableddate" type="single" [allowNull]="false"
-            placeholder="This is a disabled date picker" [formControl]="ffl5.formControl">
+            placeholder="This is a disabled date picker" [formControl]="ffl5.formControl" ariaLabel="required">
```

Comment:
https://github.com/SAP/fundamental-ngx/pull/5627 is fixing reading of required field for date-picker as well.
adding ariaLabel ="required" is not correct solution.
please see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute.